source/reference/minio-mc/mc-rm.rst

```diff
@@ -403,6 +402,10 @@ object.
 - To remove all non-current versions of an object from a bucket, use
   :mc-cmd:`mc rm --non-current`
 
+.. versionchanged:: mc RELEASE.2023-03-20T17-17-53Z
+
+   The output shows the modification time of versioned files.
+   When using ``--dry-run``, this can be helpful to determine that you select the correct object(s) for removal.
```

Comment:
Perhaps something like "When used with --dry-run, this can help confirm you are removing the correct objects."